src/chapters/angular-lsn1/components.rst

```diff
@@ -3,11 +3,11 @@ Components
 
 .. index:: ! angular-component
 
-In Angular a **component** controls a patch of screen called a view.
+In Angular a **component** controls a patch of screen called a *view*.
```

Comment:
How about controls "a part of the page called a view"?

Reply:
Rephrased as per your suggestion.  Thanks.